### PR TITLE
Add rust n-body version

### PR DESCRIPTION
**Pull request by MaxGraey**

### CODE REVIEW

File: examples/n-body/tests/index.js [hunk 1]
```diff
@@ -62,7 +66,7 @@ console.log("Took " + (time[0] * 1e3 + time[1] / 1e6) + "ms");
 console.log("\nWARMED UP:\n");
 
 console.log("Performing " + steps + " steps (AssemblyScript WASM) ...");
-time = test(nbodyWASM, steps);
+time = test(nbodyWAsmWASM, steps);
```
Comment (dcodeIO) [inline examples/n-body/tests/index.js]: WAsmWASM? :)

Comment (MaxGraey) [reply examples/n-body/tests/index.js]: I refactored this already=)

Comment (MaxGraey) [reply examples/n-body/tests/index.js]: See: https://github.com/AssemblyScript/assemblyscript/pull/88/commits/540dea56c9f6a1319933582c1da2953ddb8b318f

Comment (dcodeIO) [reply examples/n-body/tests/index.js]: Heh, what about `nbodyAS` and `nbodyRS`? :)

Comment (MaxGraey) [reply examples/n-body/tests/index.js]: Refactored to `nbodyASWasm`, `nbodyRSWasm` and `nbodyAsmJS`. WDYT?

Comment (dcodeIO) [reply examples/n-body/tests/index.js]: Hardest thing in programming? Naming things! 😅 (so, you don't like nbodyAS, nbodyRS, nbodyJS?)

Comment (MaxGraey) [reply examples/n-body/tests/index.js]: I never imagine that naming is so hard stuff 😅

Comment (MaxGraey) [reply examples/n-body/tests/index.js]: We need some philosophical conventions for naming. Something like Occam's razor=)

**Comment by dcodeIO** [comment]
Thank you ^^

**Comment by dcodeIO** [comment]
Just noticed: Can you force-add the optimized.wasm file for the Rust build, so the tests can be run without rebuilding? Did that for the AS target as well (here: in ./build).

**Pull request merged**